packages/react/src/utils/withRef.ts

```diff
@@ -0,0 +1,8 @@
+import { FC, forwardRef, ForwardRefRenderFunction } from 'react';
+
+/**
+ * Same as `forwardRef` except it returns the same type of the `component` going in.
+ * @param component Component to `forwardRef`.
```

Comment:
Cutting at 80 limit, adding space to align with all other fuction comments:

```suggestion
 * Same as `forwardRef` except it returns the same type of the `component` going
 * in.
 *
 * @param component Component to `forwardRef`.
```

Reply:
That was 81 lines 🙈 
I've updated the comment.